maplibre/src/raster/request_system.rs

```diff
@@ -48,7 +49,8 @@ impl<E: Environment, T: RasterTransferables> System for RequestSystem<E, T> {
         }: &mut MapContext,
     ) {
         let _tiles = &mut world.tiles;
-        let view_region = view_state.create_view_region();
+        let view_region =
+            view_state.create_view_region(view_state.zoom().zoom_level(DEFAULT_TILE_SIZE));
```

Comment:
not certain of the reasons -- the `.zoom().zoom_level(...)` seems weird... any way to make it into one call, like `.zoom_level(...)`?

Reply:
I could add a shorthand, not sure if that is worth it though. That call gives the zoom_level at a specific zoom. So the call makes sense even though it looks weird.